docs/imm-otel.md

```diff
@@ -0,0 +1,110 @@
+2. [Deploy an OpenTelemetry Collector for SignalFx](#deploy-an-opentelemetry-collector-for-signalfx)
+
+The following sections describe each step in detail.
+
+## Deploy the SignalFx Smart Agent
+
+The SignalFx platform is built to ingest, store, analyze, visualize, and alert on metrics at scale. A metric is anything that is measurable (you can assign a numerical value to it) and variable (changes over time). The following are examples of metrics:
+
+* CPU utilization % of a server
+* Response time in milliseconds of an API call
+* The number of unique users who logged in over the previous 24-hour period
+
+You can use the SignalFx Smart Agent to collect metrics for SignalFx Infrastructure Monitoring. See [Use the Smart Agent](https://docs.signalfx.com/en/latest/integrations/agent/index.html#smart-agent "Use the Smart Agent") for more information.
+
+**Step 1: Install the SignalFx Smart Agent**
+
+There are two options for installing the SignalFx Smart Agent:
+- Use the [quick installation](https://docs.signalfx.com/en/latest/integrations/agent/quick-install.html "Quick Install") option for simplified SignalFx Smart Agent command-line installation on a single host.
+- Use the [advanced installation](https://docs.signalfx.com/en/latest/integrations/agent/advanced-install-options.html "Advanced Installation Options") option for bulk deployments and for configuring various monitors for your environment.
+
+**Step 2: Configure the SignalFx Smart Agent**
+
+The SignalFx Smart Agent is configured primarily by a YAML document located at `/etc/signalfx/agent.yaml`. The location of the configuration file can be specified by the `-config` flag to the agent binary (`signalfx-agent`).
+
+See [Agent Configuration](https://docs.signalfx.com/en/latest/integrations/agent/config-schema.html "Agent Configuration") for complete details, including configuration options and an example configuration file.
+
+## Deploy an OpenTelemetry Collector for SignalFx
+
+After installing the SignalFx Smart Agent on each host, you can deploy an OpenTelemetry Collector in each datacenter/region/cluster where traced applications run. In general, an OpenTelemetry Collector should receive data from the SignalFx Smart Agent.
+
+The OpenTelemetry Collector uses pipelines to receive, process, and export trace data with components conveniently known as receivers, processors, and exporters. Set up pipelines with services. You can also add extensions that provide an OpenTelemetry Collector with additional functionality, such as diagnostics and health checks. The OpenTelemetry Collector has two versions: a [core version](https://github.com/open-telemetry/opentelemetry-collector "Core Version") and a [contributions version](https://github.com/open-telemetry/opentelemetry-collector-contrib "Contributions"). The core version provides receivers, processors, and exporters for general use. The contributions version provides receivers, processors, and exporters for specific vendors and use cases.
```

Comment:
it may be helpful to provide example use cases that explain why someone would deploy an otel collector between a smart agent and sfx ingest endpoint.

Reply:
I pulled some information from this specs page on use cases: https://confluence.splunk.com/display/PROD/SFx+Deliverables